cg/meta/orders/fastq_submitter.py

```diff
@@ -67,7 +67,9 @@ def create_maf_case(self, sample_obj: Sample) -> None:
             priority=Priority.research,
             ticket=sample_obj.original_ticket,
         )
-        case.customer: Customer = self.status.get_customer_by_customer_id(customer_id="cust000")
+        case.customer: Customer = self.status.get_customer_by_internal_id(
+            customer_internal_id="cust000"
```

Comment:
```suggestion
            customer_internal_id="cust000"
```
I think we have a constant for this now.

Reply:
How about "CG_INTERNAL"